packages/fiori/src/Wizard.js

```diff
@@ -706,9 +706,11 @@ class Wizard extends UI5Element {
 
 	get _steps() {
 		const lastEnabledStepIndex = this.getLastEnabledStepIndex();
+		const stepsInfo = this.getStepsInfo();
 
 		return this.steps.map((step, idx) => {
 			step.stretch = idx === lastEnabledStepIndex;
+			step.stepContentAriaLabel = `${stepsInfo[idx].number} ${stepsInfo[idx].titleText}`;
```

Comment:
The aria-label text should start with "Step"
Expected: "Step [X] <wizard-step-heading>"